compact/materials.xml

```diff
@@ -474,4 +474,26 @@
     <fraction n="0.9622" ref="Pb"/>
     <fraction n="0.0378" ref="SciFiPb_Glue"/>
   </material>
+  <material name="Brass">
+    <D type="density" value="8.41" unit="g / cm3"/>
+    <fraction n="0.70" ref="Cu"/>
+    <fraction n="0.30" ref="Zn"/>
+  </material>
+  <material name="NbTi">
+    <D type="density" value="6.54" unit="g / cm3"/>
+    <fraction n="0.50" ref="Nb"/>
+    <fraction n="0.50" ref="Ti"/>
+  </material>
+  <material name="Solder">
+    <D type="density" value="11.35" unit="g / cm3"/>
+    <fraction n="0.975" ref="Pb"/>
```

Comment:
This seems very lead-heavy for electronic solder. Is this standard solder or should we give it a more specific name so no one uses this in some maps tracker or so, thinking it's mostly tin.

Reply:
Good catch, this should be standard 60/40 electronics solder and I had specialty values. The solder definition has been updated. 

As I understood Valerio's comment the He layer is only 50um thick surrounding the coil so it was neglected in their Excel table (see the comment at the end of the first post).  The Excel table explicitly lists the inner and outer radii for each layer so we do go from 2.84-3.54m.